src/options.go

```diff
@@ -1333,6 +1354,33 @@ func parseOptions(opts *Options, allArgs []string) {
 			}
 		}
 	}
+
+	if validatePointer {
+		if err := validateSign(opts.Pointer, "pointer"); err != nil {
+			errorExit(err.Error())
+		}
+	}
+
+	if validateMarker {
+		if err := validateSign(opts.Marker, "marker"); err != nil {
+			errorExit(err.Error())
+		}
+	}
+}
+
+func validateSign(sign string, signOptName string) error {
+	if strings.Contains(sign, "\n") || strings.Contains(sign, "\t") {
```

Comment:
It would be more precise to use https://golang.org/pkg/unicode/#IsGraphic
Error message can be `invalid character in %v`